src/components/MenuItem.js

```diff
@@ -364,6 +365,11 @@ const MenuItem = React.forwardRef((props, ref) => {
                                         />
                                     </View>
                                 )}
+                                {Boolean(props.rightLabel) && (
+                                    <View style={[styles.popoverMenuIcon, styles.rightLabelMenuItem, styles.pointerEventsAuto]}>
+                                        <Text>{props.rightLabel}</Text>
```

Comment:
> Yup, we do need some style adjustments. The text should be using our textSupporting color, and it should be in our label font size (13px).

Please add text styles to meet design requirement.

Design:
<img width="377" alt="Screenshot 2023-11-19 at 10 42 26 PM" src="https://github.com/Expensify/App/assets/108292595/171c3957-0b48-48cd-8bea-7dde384fc9c9">


Current fix:
<img width="371" alt="Screenshot 2023-11-19 at 10 42 40 PM" src="https://github.com/Expensify/App/assets/108292595/b741eb9d-608e-4b12-9be4-362dcf5b19f7">